pkg/cfn/manager/api_test.go

```diff
@@ -4,20 +4,131 @@ import (
 	"errors"
 	"fmt"
 
+	"github.com/aws/aws-sdk-go-v2/service/autoscaling"
+	astypes "github.com/aws/aws-sdk-go-v2/service/autoscaling/types"
```

Comment:
```suggestion
	asTypes "github.com/aws/aws-sdk-go-v2/service/autoscaling/types"
```

small nit

Reply:
done in 83dfd587db2ddad0a41260db93b2475ce3248cb5